longest-run.playground/Contents.swift

```diff
@@ -7,23 +7,68 @@ This excercise lends itself to TDD and we have provided a few basic tests below.
 import Cocoa
 
 func longestRun(whole: String) -> [String] {
-    return [whole]
+    
+    let allRuns = stringToRunsList(whole)
+    let longestRuns = findLongestRuns(allRuns)
+    let longestRunStrings = longestRuns.map({String($0)})
+    return longestRunStrings
+}
+
+func stringToRunsList(whole: String) -> [[Character]] {
+    
+    var chars = [[Character]]()
+    
+    for index in whole.characters.indices {
+
+        let daChar = whole[index]
+        let onDaRun = chars.last?.contains(daChar)
+
+        // nil coalescing operator for da win
+        if (onDaRun ?? false)  {
```

Comment:
Definitely very cool! I prefer this syntax the ternary operator hack ?: used in C